messaging/NVSSAPI/input/fsh/NVSSAPI_CapStmt.fsh

```diff
@@ -17,4 +17,8 @@ Usage: #definition
 * rest.resource[=].interaction[0].code = #search-type
 * rest.resource[=].searchParam[0].name = "_since"
 * rest.resource[=].searchParam[=].type = #date
+* rest.resource[=].searchParam[1].name = "certificateNumber"
+* rest.resource[=].searchParam[=].type = #string
+* rest.resource[=].searchParam[2].name = "deathYear"
+* rest.resource[=].searchParam[=].type = #string
```

Comment:
Should these "=" signs be array indices? I wasn't sure why the first "_since" search param was used "=" for the type.